dnd5e.css

```diff
@@ -652,6 +652,7 @@ h5 {
   font-size: 12px;
   line-height: 20px;
   font-weight: normal;
+  align-items: baseline;
```

Comment:
I don't really know what changed here, but this fixed some alignment issues I spotted when looking at this.